<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="NE46Rs" failures="1" tests="3" name="Test" time="24.407" errors="0" timestamp="2024-09-05T13:26:58 IST">
  <testcase classname="testNg_Listeners.OrangeHrm_Test" name="test_Title" time="0.053"/>
  <testcase classname="testNg_Listeners.OrangeHrm_Test" name="test_Url" time="0.026">
    <failure type="java.lang.AssertionError" message="expected [https://opensource-demo.orangehrmlive.com/] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]">
      <![CDATA[java.lang.AssertionError: expected [https://opensource-demo.orangehrmlive.com/] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]
at testNg_Listeners.OrangeHrm_Test.test_Url(OrangeHrm_Test.java:44)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test_Url -->
  <testcase classname="testNg_Listeners.OrangeHrm_Test" name="test_Logo" time="0.0">
    <skipped/>
  </testcase> <!-- test_Logo -->
</testsuite> <!-- Test -->
